library/system/src/modules/Kernel.rb

```diff
@@ -661,6 +575,31 @@ def InformAboutKernelChange
       @inform_about_kernel_change
     end
 
+    publish function: :AddCmdLine, type: "void (string, string)"
+    publish function: :GetVgaType, type: "string ()"
+    publish function: :SetVgaType, type: "void (string)"
```

Comment:
"FIXME: is heer because of bootloader module, should be removed" and it is not used anymore!

Reply:
If you removed SetCmdLine then  SetVgaType should be removed too.